ce_clean.sh

```diff
@@ -0,0 +1,10 @@
+rm pyconfig.tmp pyconfig.pre.tmp
+rm zip.list
+rm -rf wince_build
+rm make.log
```

Comment:
ファイルがない場合にエラー終了するので `rm -rf` でまとめてください

```suggestion
rm -rf pyconfig.tmp pyconfig.pre.tmp zip.list wince_build make.log
```

Reply:
了解です。この`ce_clean.sh`はまだかなり適当なので、もう少しきちんと書き直したほうがいいかもしれないですね...